docs/source/_static/getting-started.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
     "import numpy as np\n",
     "\n",
-    "data = np.genfromtxt('../data/exampleData.csv', delimiter=',')\n",
+    "data = np.genfromtxt('exampleData.csv', delimiter=',')\n",
```

Comment:
Let's modify this block to show off our preprocessing functionality for reading in data and filtering out data points with positive imaginary parts. Most of the other example notebooks use the numpy approach seen here, but I can quickly change those and add a commit to your PR.

Reply:
I'm having some trouble adding my commit to this PR. Is the "Allow edits from maintainers." box selected?

Reply:
Huh weird @BGerwe, I have it checked on the PR here:
![image](https://user-images.githubusercontent.com/29909203/80895553-12b6a280-8c9b-11ea-909f-8b90459b9c64.png)

Are you committing on the user fork or the ECSHackWeek repository? I believe it needs to be on my user fork

Reply:
I got it to work this morning, I think I was misspelling something in the push command yesterday... silly. Anyways, this looks great!